SubmittedAssignments/sanjeevanee_balure_66/com..sty..css

```
@@ -0,0 +1,149 @@
+			index: z-index;
+		}
+		.certificate .recttop2 {
+		position: absolute;
+		    bottom: 0pt;
+			right: 0pt;
+            width: 30pt;
+            height: 179pt;
+            background-color: #002147;
+			border: 2pt solid goldenrod;
+            border-radius: 20pt 0pt 0pt 0pt;
+			index: x-index;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)